src/mount.c

```diff
@@ -152,6 +153,17 @@ cc_oci_perform_mount (const struct cc_oci_mount *m, gboolean dry_run)
 	if (dry_run) {
 		return true;
 	}
+	if (g_file_test (m->mnt.mnt_fsname, G_FILE_TEST_IS_REGULAR)) {
+		struct stat st;
+		ret = stat (m->mnt.mnt_fsname, &st);
+		if (! ret) {
+			return false;
+		}
+		ret = creat (m->dest, st.st_mode);
```

Comment:
I was going to say that there needs to be a corresponding `remove()` call in `cc_oci_perform_unmount()`, but I see that we don't do that for directories either. Hence, this PR is consistent with the existing code.

(I've created issue #178 to handle general mount cleanup).